src/taskCard.js

```diff
@@ -0,0 +1,24 @@
+import './mainContainer.css';
+import editBtn from './image/edit.svg'
+
+const TaskCard = (props) =>{
```

Comment:
```suggestion

const TaskCard = ({ title, display, description }) =>{
```
You may use destructuring of `props`, to avoid typing again & again `props.title`, etc